CSE437s/FL24-project-group-9
# Add Redis for backend

# Description

- Add in-memory cache for backend

## Related PRs

#88 

## Type of change

### Please delete options that are not relevant


## Screenshots/Recordings

n/a

## Testing instructions

Instructions on how to test the changes made in the PR, helping reviewers validate the code.

1. Run `git checkout branch_name`.
2. Check all available features and endpoints.

User [loadtest](https://www.npmjs.com/package/loadtest) to test redis performance.
## Notes

If there are any specific instructions or considerations you want to highlight for the reviewers, include them in this section.
